newsfragments/2825.removal.rst

```diff
@@ -0,0 +1 @@
+Now setuptools declares its own dependencies in the ``core`` extra. Dependencies are still vendored for bootstrapping purposes, but installed dependencies are preferred. Downstream packagers can de-vendor by simply removing the ``setuptools/_vendor`` directory.
```

Comment:
Should we change this changelog entry?

Specifically the phrase "but installed dependencies are preferred" is a bit puzzling to me for the following reason:

* It seems to indicate that we want users to write:
   ```toml
   [build-system]
   requires = [
      "setuptools[core]"
   ]
   ...
   ```
   But we know that this can cause cycles and therefore clashes with PEP 517... So I don't think we should recommend that...

Due to the limitations that we have right now `core` can only be "informational" for the time being (but not intended for general use).